<launch>
    <!-- joy node -->
  <node respawn="true" pkg="joy" type="joy_node" name="joy" output="screen">    
  	<param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

	<node name="PlannerNode" pkg="turtlebot_nav" type="PlannerNode.py" output="screen" required="true"/>
	<node name="SLPredictor" pkg="turtlebot_nav" type="SLPredictor.py" output="screen" required="true"/>

	<node name="turtlebot_nav" pkg="turtlebot_nav" type="turtlebot_nav" output="screen" required="true">
		<param name="qThresh" type="double" value="-0.2" />	
		<param name="mode" type="string" value="TRAIN" />
		<param name="num_episodes" type="int" value="20" />
		<param name="max_steps" type="int" value="1000" />
		<param name="init_x" type="double" value="-1" />
		<param name="init_y" type="double" value="1" />
		<param name="init_Y" type="double" value="0.4" />
	</node>

	
	
</launch>
